include/net/nrf_cloud.h

```diff
@@ -645,13 +650,13 @@ int nrf_cloud_send(const struct nrf_cloud_tx_data *msg);
  *
  * This API may be called any time after receiving the
  * @ref NRF_CLOUD_EVT_TRANSPORT_CONNECTED event.
- * If the API succeeds, you can expect the
- * @ref NRF_CLOUD_EVT_TRANSPORT_DISCONNECTED event.
+ * The @ref NRF_CLOUD_EVT_TRANSPORT_DISCONNECTED event indicates
+ * that the disconnect has completed successfully.
  *
  * @retval 0       If successful.
  * @retval -EACCES Cloud connection is not established; wait for
  *                 @ref NRF_CLOUD_EVT_TRANSPORT_CONNECTED.
- *                 Otherwise, a (negative) error code is returned.
+ * @return A negative value indicates an error.
```

Comment:
What is the effect of having both return and retval? You do this in quite a few, so I am assuming it is intentional, but I don't understand it

Reply:
you can see examples of both:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/networking/nrf_cloud.html

this ways looks better imo, but i guess @peknis has the final say.